Source/Scene/Cesium3DTileFeature.js

```diff
@@ -0,0 +1,270 @@
+     * modifications. Also listen to the {@link Cesium3DTileset#tileVisible} event to reapply any modifications.
+     * </p>
+     * <p>
+     * Do not construct this directly.  Access it through {@link Cesium3DTileContent#getFeature}
+     * or picking using {@link Scene#pick} and {@link Scene#pickPosition}.
+     * </p>
+     *
+     * @alias Cesium3DTileFeature
+     * @constructor
+     *
+     * @example
+     * // On mouse over, display all the properties for a feature in the console log.
+     * handler.setInputAction(function(movement) {
+     *     var feature = scene.pick(movement.endPosition);
+     *     if (feature instanceof Cesium.Cesium3DTileFeature) {
+     *         var propertyNames = feature.getPropertyNames();
+     *         var length = propertyNames.length;
+     *         for (var i = 0; i < length; ++i) {
+     *             var propertyName = propertyNames[i];
+     *             console.log(propertyName + ': ' + feature.getProperty(propertyName));
+     *         }
+     *     }
+     * }, Cesium.ScreenSpaceEventType.MOUSE_MOVE);
+     */
+    function Cesium3DTileFeature(tileset, content, batchId) {
+        this._content = content;
+        this._batchId = batchId;
+        this._color = undefined;  // for calling getColor
+    }
+
+    defineProperties(Cesium3DTileFeature.prototype, {
+        /**
+         * Gets or sets if the feature will be shown. This is set for all features
+         * when a style's show is evaluated.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Boolean}
+         *
+         * @default true
+         */
+        show : {
+            get : function() {
+                return this._content.batchTable.getShow(this._batchId);
+            },
+            set : function(value) {
+                this._content.batchTable.setShow(this._batchId, value);
+            }
+        },
+
+        /**
+         * Gets or sets the highlight color multiplied with the feature's color.  When
+         * this is white, the feature's color is not changed. This is set for all features
+         * when a style's color is evaluated.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Color}
+         *
+         * @default {@link Color.WHITE}
+         */
+        color : {
+            get : function() {
+                if (!defined(this._color)) {
+                    this._color = new Color();
+                }
+                return this._content.batchTable.getColor(this._batchId, this._color);
+            },
+            set : function(value) {
+                this._content.batchTable.setColor(this._batchId, value);
+            }
+        },
+
+        /**
+         * Gets the content of the tile containing the feature.
+         *
+         * @memberof Cesium3DTileFeature.prototype
+         *
+         * @type {Cesium3DTileContent}
+         *
+         * @readonly
+         * @private
```

Comment:
Hi, 
we would like to have api access to the original batchTableJson. For this it would be good if the @private here and at https://github.com/AnalyticalGraphicsInc/cesium/blob/3d-tiles/Source/Scene/Cesium3DTileContent.js#L208
could be removed. Or maybe a direct accessor to the BatchTableJson in the Cesium3DTileContent.js?